config/src/test/resources/CustomJdbcUserServiceSampleConfig.sql

```diff
@@ -1,4 +1,4 @@
-create table users(principal varchar_ignorecase(50) not null primary key,credentials varchar_ignorecase(50) not null);
+create table users(principal varchar_ignorecase(50) not null primary key,credentials varchar_ignorecase(100) not null);
```

Comment:
Increased size to allow the test to succeed - the encoded password is longer than 50 characters.